apps/surfer/.gitignore

```diff
@@ -0,0 +1 @@
+!.env.production
```

Comment:
Do we have to keep it in the repo?

Reply:
I wanted to do it this way, instead of hardcoding the URL. This way it's used during build, but can easily be overriden locally.